src/it/resources/com/google/checkstyle/test/chapter5naming/rule525nonconstantfieldnames/InputNonConstantNamesSimple.java

```diff
@@ -36,20 +36,8 @@ final class InputNonConstantNamesSimple {
 
   public int mTes$t2; // violation 'Member name 'mTes\$t2' must match pattern'
 
-  /** test local variables. This Check doesn't verify local variables, only members. */
   private void localVariables() {
     int a;
-    int aA;
```

Comment:
removed to avoid unrelated violations `// violation 'Local variable name 'aA' must match pattern'`